spec/requests/incoming_webhooks/mailchimp/unsubscribe_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+RSpec.describe "IncomingWebhooks::MailchimpUnsubscribesController", type: :request do
+  let(:user) { create(:user, email_digest_periodic: true) }
+
+  describe "POST /webhooks/mailchimp/:secret/unsubscribe" do
```

Comment:
there's a typo here but I don't think it should hold the PR.

Reply:
Do you mean that it should say `incoming_webhooks`? Because I don't really see a typo, but that could also be because I knew what I wanted to write 😉